example/fields.js

```diff
@@ -26,7 +26,8 @@ module.exports = {
     placeholder: "Student or professional at ...",
     validator: {
       error: "Please let us know what your occupation is."
-    }
+    },
+    wordLimit: 30
```

Comment:
We should update the "custom 51 character field" to use the charlimit functionality, as well. Right now it's hardcoded in the app, which is bad.